tests/test_ghosting.py

```diff
@@ -40,16 +44,18 @@ def test_calculate_ghost_intensity(self):
 
         with pytest.raises(Exception):
             hazen_ghosting.calculate_ghost_intensity(ghost=np.asarray([-10]),
-                                                     phantom=np.asarray([-100]),
-                                                     noise=np.asarray([-5]))
+             phantom=np.asarray([-100]),
+         noise=np.asarray([-5]))
 
         assert 5.0 == hazen_ghosting.calculate_ghost_intensity(ghost=np.asarray([10]),
-                                                               phantom=np.asarray([100]),
-                                                               noise=np.asarray([5]))
+                                               phantom=np.asarray([100]),
+                                        noise=np.asarray([5]))
```

Comment:
Some of the indentation in this section is awry – would you mind just tidying.

Reply:
I'll tidy up the indentation.